### DISCUSSION TITLE
move server-build/index.js to build/server/index.js

### DISCUSSION THREAD

**Discussion by onemen** [epicweb-dev/epic-stack#135] [Ideas]
@kentcdodds 
currently `server-build/index.js` is out of the build folder.

I think it is better to move it to `build/server/index.js`
we can use code similar to [this](https://github.com/epicweb-dev/kcdshop/blob/main/packages/workshop-app/other/build-dir.ts).

**Comment by kentcdodds** [comment]
The problem with doing this is that the relative import of trying to import something in a parent directory changes which is surprising and would break, but only in production mode. The build needs to be the same distance from the parent directory as the source.

**Reply by onemen** [reply]
https://github.com/epicweb-dev/kcdshop/blob/714bf269b5c41e4c92eec9eb5390682c80662dd2/packages/workshop-app/other/build-dir.ts#L55-L60

i've change the import with esbuild from '../build/index.js' to '../index.js'
```js
                        if (dir === 'server' && /server[\\/]index\.ts/.test(args.path)) {
                                contents = contents.replace(
                                        RegExp('../build/index.js', 'g'),
                                        '../index.js',
                                )
                        }
```

**Reply by kentcdodds** [reply]
Sure, but if someone were to put a config file at the root and then import it, that would work in development but not in production.

**Reply by onemen** [reply]
I understand
